[CS2103T-F13-4] ezwatchlist

heze8 opened the pull request:
@jcjjjared
@michelleykw
@tswuxia
@wongchuankai

jcjjjared committed:
Create empty text file to commit to test travis

jcjjjared committed:
Update the link to the 'build status' badge at the top of the README.adoc to point to the build status of own repo

jcjjjared committed:
Adjust README.adoc file to point to  AY1920S1-CS2103T-F13-4 / main

jcjjjared committed:
Remove the codacy badge from README.adoc

jcjjjared committed:
Correct README.adoc coverage and status badges

jcjjjared committed:
Added AboutUs.adoc and Jared's photo

jcjjjared committed:
Updated ContactUs.adoc to reference to our own project

jcjjjared committed:
Merge branch 'master' of https://github.com/jcjjjared/main

heze8 committed:
no message

jcjjjared committed:
Removed trailing whitespace in AboutUs.adoc

heze8 committed:
fixed aboutUs with travis

heze8 committed:
Merge pull request #6 from heze8/master

Updated AboutUs.doc

michelleykw committed:
Update ReadMe

michelleykw committed:
Merge branch 'branch-readme'

* branch-readme:
  Update ReadMe

michelleykw committed:
Merge pull request #8 from michelleykw/branch-readme

Update ReadMe

jcjjjared committed:
Resolve merge conflicts

michelleykw committed:
Update Ui image

jcjjjared committed:
Merge pull request #4 from jcjjjared/master

Updated ContactUs.adoc to reference to our own project

michelleykw committed:
Merge branch 'branch-readme'

* branch-readme:
  Update Ui image

michelleykw committed:
Merge pull request #10 from michelleykw/branch-readme

Update Ui image

wongchuankai committed:
updated devloper guide

heze8 committed:
Merge pull request #11 from AY1920S1-CS2103T-F13-4/about-me

Developer guide

michelleykw committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-F13-4/main

* 'master' of https://github.com/AY1920S1-CS2103T-F13-4/main:
  updated devloper guide
  Resolve merge conflicts
  fixed aboutUs with travis
  Removed trailing whitespace in AboutUs.adoc
  no message
  Updated ContactUs.adoc to reference to our own project
  Added AboutUs.adoc and Jared's photo

michelleykw committed:
Fixed error in Ui image

michelleykw committed:
Merge pull request #12 from michelleykw/master

Fixed error in Ui image

wongchuankai committed:
Edited Developer Guide
1) changed site-section: UserGuide -> DeveloperGuide
2) Added Value proposition

wongchuankai committed:
Merge pull request #13 from AY1920S1-CS2103T-F13-4/about-me

Edited Developer Guide

jcjjjared committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-F13-4/main

jcjjjared committed:
Updated user guide and included images for it

jcjjjared committed:
Merge branch 'branch-User-Guide'

jcjjjared committed:
Merge pull request #15 from jcjjjared/master

Update user guide

jcjjjared committed:
Fix user guide formatting issue

jcjjjared committed:
Merge branch 'branch-User-Guide'

jcjjjared committed:
Merge pull request #16 from jcjjjared/master

Fix formatting issues with user guide

heze8 committed:
Added non-functionality requirements in the developer guide

heze8 committed:
Merge pull request #17 from heze8/master

Added non-functionality requirements in the developer guide

wongchuankai committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-F13-4/addressbook-level3

tswuxia committed:
Updated use cases in developerGuide.adoc

michelleykw committed:
Updated glossary in DeveloperGuide.adoc

michelleykw committed:
Merge pull request #22 from michelleykw/master

Updated glossary in DeveloperGuide.adoc

michelleykw committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-F13-4/main

* 'master' of https://github.com/AY1920S1-CS2103T-F13-4/main:

michelleykw committed:
Update glossary in DeveloperGuide.adoc

junnbang commented on docs/DeveloperGuide.adoc:
```diff
@@ -367,33 +367,140 @@ _{More to be added}_
+*MSS*
+
+1.  User navigates to the page containing the item to be deleted.
+2.  User enters 'delete' command giving the name of the item.
```
Can be more high level language / abstract. Can use "request" instead of saying how the GUI interacts with the user.

tswuxia committed:
Changed one line of use cases

michelleykw committed:
Merge pull request #30 from michelleykw/master

Update DeveloperGuide.adoc

jcjjjared committed:
Create structure for the classes

jcjjjared committed:
Edited Show, Movie, TvShow, TvSeason and Episode classes

jcjjjared committed:
Included UniqueShowList, DuplicateShowException and ShowNotFoundException

jcjjjared committed:
Organise the structure of the model package

jcjjjared committed:
Removed trailling whitespaces

jcjjjared committed:
Added javadocs for come classes

jcjjjared committed:
Fix lexicographical order of imports

jcjjjared committed:
Fix lexicographical order of imports again

jcjjjared committed:
Merge pull request #50 from jcjjjared/branch-v1.2

Branch v1.2

jcjjjared committed:
Added model classes for watchlist

heze8 committed:
Merge branch 'branch-v1.2' of https://github.com/AY1920S1-CS2103T-F13-4/main

heze8 committed:
Added the Api into the application
added a placeholder image field in the show model class
Added one method in Api Main

heze8 committed:
Added tv show method
things to take note, the show model might not need all the variables in the constructor at creation, maybe can replace it with the required information first, then another method to check if all the information can be taken from the api
images are also anothher thing that might be quite difficult to handle. Should we retrieve images online and download into resources so that it will be cached?

heze8 committed:
no message

jcjjjared committed:
Reduce travis checks

tswuxia committed:
Merge pull request #54 from heze8/branch-api

Branch api

jcjjjared committed:
no message

jcjjjared committed:
no message

jcjjjared committed:
Merge pull request #55 from jcjjjared/master

Reduce travis checks

jcjjjared committed:
Test if travis chekcs have been reduced by removing lastline

jcjjjared committed:
Restore travis checks

jcjjjared committed:
Merge pull request #57 from jcjjjared/master

Restore travis.yml

heze8 committed:
Merge pull request #31 from tswuxia/master

Changed one line of use cases

heze8 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-F13-4/main into branch-api

heze8 committed:
Merge pull request #51 from jcjjjared/branch-v1.2

Added model classes for watchlist

heze8 committed:
Merge branch Jared's 'branch-v1.2' of https://github.com/AY1920S1-CS2103T-F13-4/main into my branch-api

heze8 committed:
Added rudimentary spellcheck classes for Michelle to take a look
Added one MovieCard that has the same format as PersonCard so that we can start working on the interface changes
Tweaked some of the API methods
Refactored the name of the package from adress to ezwatchlist
Added a movieInfromation class in the api package but might be unnecesarry. Could perhaps change the in-house movie class to be similar to that

heze8 committed:
Error still exist can't seem to find the interface

jcjjjared committed:
Merge pull request #65 from heze8/branch-caleb-v1.2

Branch caleb v1.2

jcjjjared committed:
Revert "Branch caleb v1.2"

jcjjjared committed:
Merge pull request #66 from AY1920S1-CS2103T-F13-4/revert-65-branch-caleb-v1.2

Revert "Branch caleb v1.2"
